src/html/openstack_accounting/main.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- prepared using https://datatables.net/download/ -->
+    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/dt/jq-3.6.0/jszip-2.5.0/dt-1.12.1/b-2.2.3/b-colvis-2.2.3/b-html5-2.2.3/b-print-2.2.3/cr-1.5.6/date-1.1.2/kt-2.7.0/rr-1.2.8/sc-2.0.6/sb-1.3.3/sp-2.0.1/sl-1.4.0/sr-1.1.1/datatables.min.css"/>
+
+    <!--  Please do not delete below CSSes, important for pretty view -->
+    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/dt/dt-1.12.1/datatables.min.css"/>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdn.datatables.net/1.11.4/css/dataTables.bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdn.datatables.net/buttons/2.2.2/css/buttons.bootstrap.min.css">
+
+    <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
+    <style>
+        body {
+	        font-family: 'Trebuchet MS', sans-serif;
+        }
+        /* Search bar */
+        .dataTables_filter input {
+          border: 7px solid Tomato;
+          width: 400px;
+          font-size: 14px;
+          font-weight: bold;
+        }
+        table td {
+            word-break: break-all;
+        }
```

Comment:
When you separate column words with space, this setting will make the trick :
```
table th {
          white-space: wrap;
        }
```

Reply:
Can you please check, if in my next commit, this will be correctly fixed? Just to be 100% sure. Thank you